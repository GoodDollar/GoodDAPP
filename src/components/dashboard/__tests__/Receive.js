// eslint-disable-next-line import/order
import React from 'react'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

jest.setTimeout(30000)

let getWebRouterComponentWithMocks
describe('Receive', () => {
  beforeAll(() => {
    localStorage.setItem(
      'GD_USER_MNEMONIC',
      'burger must derive wrong dry unaware reopen laptop acoustic report slender scene',
    )
    getWebRouterComponentWithMocks = require('./__util__').getWebRouterComponentWithMocks
  })

  it('matches snapshot', async () => {
    const Receive = getWebRouterComponentWithMocks('../Receive')
    let component

    // eslint-disable-next-line require-await
    await renderer.act(async () => (component = renderer.create(<Receive />)))
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
