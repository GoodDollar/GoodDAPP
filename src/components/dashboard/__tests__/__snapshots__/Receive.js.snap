// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Receive matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n"
  style={
    Object {
      "WebkitBoxFlex": 1,
      "WebkitFlexBasis": "0%",
      "WebkitFlexGrow": 1,
      "WebkitFlexShrink": 1,
      "flexBasis": "0%",
      "flexGrow": 1,
      "flexShrink": 1,
      "msFlexNegative": 1,
      "msFlexPositive": 1,
      "msFlexPreferredSize": "0%",
    }
  }
>
  <div
    className="css-view-1dbjc4n"
    data-name="viewWrapper"
    style={
      Object {
        "WebkitBoxDirection": "normal",
        "WebkitBoxFlex": 1,
        "WebkitBoxOrient": "vertical",
        "WebkitFlexDirection": "column",
        "WebkitFlexGrow": 1,
        "backgroundImage": "linear-gradient(to bottom, #00AFFF, #2DC0F7, #28C0EF, #23C0E7, #1EC1DF, #19C1D7, #14C1CF, #0FC2C7, #0FC2C7, #0AC2BF, #05C2B7, #00C3AF)",
        "display": "flex",
        "flexDirection": "column",
        "flexGrow": 1,
        "maxHeight": "844px",
        "msFlexDirection": "column",
        "msFlexPositive": 1,
        "paddingBottom": "8px",
        "paddingLeft": "8px",
        "paddingRight": "8px",
        "paddingTop": "8px",
        "position": "relative",
        "width": "100%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxPack": "center",
          "WebkitJustifyContent": "center",
          "backgroundColor": "rgba(255,255,255,1.00)",
          "borderBottomLeftRadius": "5px",
          "borderBottomRightRadius": "5px",
          "borderTopLeftRadius": "5px",
          "borderTopRightRadius": "5px",
          "height": "62px",
          "justifyContent": "center",
          "marginBottom": "8px",
          "msFlexPack": "center",
          "paddingBottom": "8px",
          "paddingLeft": "12px",
          "paddingRight": "8px",
          "paddingTop": "8px",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxDirection": "reverse",
            "WebkitBoxOrient": "horizontal",
            "WebkitBoxPack": "justify",
            "WebkitFlexDirection": "row-reverse",
            "WebkitJustifyContent": "space-between",
            "alignItems": "center",
            "flexDirection": "row-reverse",
            "justifyContent": "space-between",
            "msFlexAlign": "center",
            "msFlexDirection": "row-reverse",
            "msFlexPack": "justify",
          }
        }
      >
        <div
          className="css-view-1dbjc4n"
        />
        <div
          className="css-view-1dbjc4n"
          data-focusable={true}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "MozTransitionProperty": "opacity",
              "MozUserSelect": "none",
              "WebkitTransitionDuration": "0.15s",
              "WebkitTransitionProperty": "opacity",
              "WebkitUserSelect": "none",
              "cursor": "pointer",
              "msTouchAction": "manipulation",
              "msUserSelect": "none",
              "touchAction": "manipulation",
              "transitionDuration": "0.15s",
              "transitionProperty": "opacity",
              "userSelect": "none",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "alignItems": "center",
                "display": "flex",
                "marginLeft": "20px",
                "msFlexAlign": "center",
              }
            }
          >
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "WebkitAlignItems": "center",
                  "WebkitBoxAlign": "center",
                  "WebkitBoxPack": "center",
                  "WebkitJustifyContent": "center",
                  "alignItems": "center",
                  "backgroundColor": "rgba(12,38,61,1.00)",
                  "borderBottomLeftRadius": "21px",
                  "borderBottomRightRadius": "21px",
                  "borderTopLeftRadius": "21px",
                  "borderTopRightRadius": "21px",
                  "display": "flex",
                  "height": "30px",
                  "justifyContent": "center",
                  "marginBottom": "5px",
                  "msFlexAlign": "center",
                  "msFlexPack": "center",
                  "width": "30px",
                }
              }
            >
              <div
                className="css-text-901oao"
                dir="auto"
                style={
                  Object {
                    "color": "rgba(255,255,255,1.00)",
                    "fontFamily": "gooddollar",
                    "fontSize": "20px",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  }
                }
              >
                î¡ƒ
              </div>
            </div>
            <div
              className="css-text-901oao"
              dir="auto"
              style={
                Object {
                  "color": "rgba(12,38,61,1.00)",
                  "direction": "ltr",
                  "fontFamily": "Roboto",
                  "fontSize": "10px",
                  "fontWeight": "500",
                  "lineHeight": "11px",
                  "textAlign": "center",
                  "textDecoration": "none",
                  "textTransform": "none",
                }
              }
            >
              Receive to address
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitBoxFlex": 1,
          "WebkitFlexGrow": 1,
          "backgroundColor": "rgba(255,255,255,1.00)",
          "borderBottomLeftRadius": "5px",
          "borderBottomRightRadius": "5px",
          "borderTopLeftRadius": "5px",
          "borderTopRightRadius": "5px",
          "flexGrow": 1,
          "msFlexPositive": 1,
          "paddingBottom": "16px",
          "paddingLeft": "12px",
          "paddingRight": "12px",
          "paddingTop": "16px",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "height": "25%",
          }
        }
      />
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxDirection": "normal",
            "WebkitBoxOrient": "vertical",
            "WebkitBoxPack": "center",
            "WebkitFlexDirection": "column",
            "WebkitJustifyContent": "center",
            "alignItems": "center",
            "flexDirection": "column",
            "justifyContent": "center",
            "msFlexAlign": "center",
            "msFlexDirection": "column",
            "msFlexPack": "center",
          }
        }
      >
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "color": "rgba(66,69,74,1.00)",
              "direction": "ltr",
              "fontFamily": "Roboto",
              "fontSize": "16px",
              "fontWeight": "500",
              "lineHeight": "22px",
              "marginBottom": "24px",
              "textAlign": "center",
              "textDecoration": "none",
              "textTransform": "none",
            }
          }
        >
          Let someone scan your wallet address
        </div>
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "WebkitAlignItems": "center",
              "WebkitBoxAlign": "center",
              "WebkitBoxPack": "center",
              "WebkitJustifyContent": "center",
              "alignItems": "center",
              "justifyContent": "center",
              "msFlexAlign": "center",
              "msFlexPack": "center",
            }
          }
        >
          <div
            className="css-view-1dbjc4n"
            style={
              Object {
                "borderBottomColor": "rgba(0,175,255,1.00)",
                "borderBottomLeftRadius": "5px",
                "borderBottomRightRadius": "5px",
                "borderBottomWidth": "1px",
                "borderLeftColor": "rgba(0,175,255,1.00)",
                "borderLeftWidth": "1px",
                "borderRightColor": "rgba(0,175,255,1.00)",
                "borderRightWidth": "1px",
                "borderTopColor": "rgba(0,175,255,1.00)",
                "borderTopLeftRadius": "5px",
                "borderTopRightRadius": "5px",
                "borderTopWidth": "1px",
                "paddingBottom": "16px",
                "paddingLeft": "16px",
                "paddingRight": "16px",
                "paddingTop": "16px",
              }
            }
          >
            <canvas
              height={128}
              style={
                Object {
                  "height": 128,
                  "width": 128,
                }
              }
              theme={
                Object {
                  "backdrop": Object {
                    "backgroundColor": "rgba(255, 0, 0, 0.8)",
                  },
                  "borders": Object {
                    "defaultBorderColor": "#c9c8c9",
                  },
                  "colors": Object {
                    "accent": "#03dac4",
                    "backdrop": "rgba(0, 0, 0, 0.5)",
                    "background": "#f6f6f6",
                    "blue": "#006EA0",
                    "darkBlue": "#0C263D",
                    "darkGray": "#42454A",
                    "disabled": "#E3E3E2",
                    "error": "#FA6C77",
                    "facebookBlue": "#1877F3",
                    "googleRed": "#EA4335",
                    "gray": "#555",
                    "gray50Percent": "#CBCBCB",
                    "gray80Percent": "#A3A3A3",
                    "green": "#00C3AE",
                    "lightGray": "#EEE",
                    "lightGreen": "#00BBA5",
                    "lighterGray": "#A3A3A3",
                    "notification": "#f50057",
                    "orange": "#F8AF40",
                    "placeholder": "#CBCBCB",
                    "primary": "#00AFFF",
                    "purple": "#9F6A9D",
                    "red": "#FA6C77",
                    "surface": "#ffffff",
                    "text": "#222",
                    "white": "#ffffff",
                  },
                  "dark": false,
                  "feedItems": Object {
                    "borderRadius": 8,
                    "height": 84,
                    "itemBackgroundColor": "#fff",
                  },
                  "fontStyle": Object {
                    "color": "#555",
                    "fontSize": 18,
                    "textAlign": "center",
                  },
                  "fonts": Object {
                    "default": "Roboto",
                    "light": "Roboto-Light, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                    "medium": "Roboto-Medium, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                    "regular": "Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                    "slab": "Roboto Slab",
                    "thin": "Roboto-Thin, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                  },
                  "modals": Object {
                    "activityIndicatorBackgroundColor": "rgba(255, 255, 255, 0.8)",
                    "backgroundColor": "#fff",
                    "borderLeftWidth": 10,
                    "borderRadius": 5,
                    "contentPadding": 16,
                    "jaggedEdgeSize": 16,
                    "overlayBackgroundColor": "transparent",
                    "overlayHorizontalPadding": 20,
                    "overlayVerticalPadding": 65,
                  },
                  "paddings": Object {
                    "defaultMargin": 8,
                    "mainContainerPadding": 8,
                  },
                  "roundness": 4,
                  "sizes": Object {
                    "borderRadius": 5,
                    "default": 8,
                    "defaultDouble": 16,
                    "defaultHalf": 4,
                    "defaultQuadruple": 32,
                    "maxHeightForTabletAndDesktop": 844,
                    "maxWidthForTabletAndDesktop": 475,
                    "minHeightForDialogMessage": 350,
                  },
                }
              }
              width={128}
            />
          </div>
        </div>
      </div>
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxDirection": "normal",
            "WebkitBoxFlex": 1,
            "WebkitBoxOrient": "vertical",
            "WebkitBoxPack": "center",
            "WebkitFlexDirection": "column",
            "WebkitFlexGrow": 1,
            "WebkitJustifyContent": "center",
            "alignItems": "center",
            "flexDirection": "column",
            "flexGrow": 1,
            "justifyContent": "center",
            "marginBottom": "20px",
            "marginTop": "20px",
            "msFlexAlign": "center",
            "msFlexDirection": "column",
            "msFlexPack": "center",
            "msFlexPositive": 1,
          }
        }
      >
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "color": "rgba(66,69,74,1.00)",
              "direction": "ltr",
              "fontFamily": "Roboto",
              "fontSize": "14px",
              "fontWeight": "normal",
              "lineHeight": "20px",
              "textAlign": "center",
              "textDecoration": "none",
              "textTransform": "none",
            }
          }
        >
          - OR -
        </div>
      </div>
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "stretch",
            "WebkitBoxAlign": "stretch",
            "WebkitBoxDirection": "normal",
            "WebkitBoxOrient": "vertical",
            "WebkitBoxPack": "justify",
            "WebkitFlexDirection": "column",
            "WebkitJustifyContent": "space-between",
            "alignItems": "stretch",
            "flexDirection": "column",
            "justifyContent": "space-between",
            "msFlexAlign": "stretch",
            "msFlexDirection": "column",
            "msFlexPack": "justify",
          }
        }
      >
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "WebkitAlignItems": "stretch",
              "WebkitBoxAlign": "stretch",
              "WebkitBoxPack": "center",
              "WebkitJustifyContent": "center",
              "alignItems": "stretch",
              "backgroundColor": "rgba(0,0,0,0.00)",
              "borderBottomColor": "rgba(0,175,255,1.00)",
              "borderBottomLeftRadius": "50px",
              "borderBottomRightRadius": "50px",
              "borderBottomStyle": "solid",
              "borderBottomWidth": "1px",
              "borderLeftColor": "rgba(0,175,255,1.00)",
              "borderLeftStyle": "solid",
              "borderLeftWidth": "1px",
              "borderRightColor": "rgba(0,175,255,1.00)",
              "borderRightStyle": "solid",
              "borderRightWidth": "1px",
              "borderTopColor": "rgba(0,175,255,1.00)",
              "borderTopLeftRadius": "50px",
              "borderTopRightRadius": "50px",
              "borderTopStyle": "solid",
              "borderTopWidth": "1px",
              "display": "flex",
              "justifyContent": "center",
              "marginBottom": "0px",
              "marginLeft": "0px",
              "marginRight": "0px",
              "marginTop": "0px",
              "minHeight": "44px",
              "minWidth": "64px",
              "msFlexAlign": "stretch",
              "msFlexPack": "center",
              "paddingBottom": "0px",
              "paddingLeft": "0px",
              "paddingRight": "0px",
              "paddingTop": "0px",
            }
          }
        >
          <div
            className="css-cursor-18t94o4 css-view-1dbjc4n"
            data-focusable={true}
            disabled={false}
            onKeyDown={[Function]}
            onKeyPress={[Function]}
            onKeyUp={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            role="button"
            style={
              Object {
                "borderBottomLeftRadius": "50px",
                "borderBottomRightRadius": "50px",
                "borderTopLeftRadius": "50px",
                "borderTopRightRadius": "50px",
                "cursor": "pointer",
                "msTouchAction": "manipulation",
                "overflowX": "hidden",
                "overflowY": "hidden",
                "position": "relative",
                "touchAction": "manipulation",
              }
            }
            tabIndex="0"
          >
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "WebkitAlignItems": "center",
                  "WebkitBoxAlign": "center",
                  "WebkitBoxDirection": "normal",
                  "WebkitBoxOrient": "horizontal",
                  "WebkitBoxPack": "center",
                  "WebkitFlexDirection": "row",
                  "WebkitJustifyContent": "center",
                  "alignItems": "center",
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "msFlexAlign": "center",
                  "msFlexDirection": "row",
                  "msFlexPack": "center",
                }
              }
            >
              <div
                className="css-text-901oao css-textOneLine-bfa6kz"
                dir="auto"
                style={
                  Object {
                    "color": "rgba(0,175,255,1.00)",
                    "direction": "ltr",
                    "fontFamily": "Roboto-Medium, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                    "letterSpacing": "1px",
                    "marginBottom": "9px",
                    "marginLeft": "16px",
                    "marginRight": "16px",
                    "marginTop": "9px",
                    "textAlign": "center",
                  }
                }
              >
                <div
                  className="css-view-1dbjc4n"
                  style={
                    Object {
                      "WebkitAlignItems": "center",
                      "WebkitBoxAlign": "center",
                      "WebkitBoxDirection": "normal",
                      "WebkitBoxFlex": 1,
                      "WebkitBoxOrient": "horizontal",
                      "WebkitFlexBasis": "0%",
                      "WebkitFlexDirection": "row",
                      "WebkitFlexGrow": 1,
                      "WebkitFlexShrink": 1,
                      "alignItems": "center",
                      "display": "flex",
                      "flexBasis": "0%",
                      "flexDirection": "row",
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "msFlexAlign": "center",
                      "msFlexDirection": "row",
                      "msFlexNegative": 1,
                      "msFlexPositive": 1,
                      "msFlexPreferredSize": "0%",
                    }
                  }
                >
                  <span
                    className="css-text-901oao css-textHasAncestor-16my406"
                    dir="auto"
                    style={
                      Object {
                        "WebkitAlignItems": "center",
                        "WebkitBoxAlign": "center",
                        "WebkitBoxPack": "center",
                        "WebkitJustifyContent": "center",
                        "alignItems": "center",
                        "color": "rgba(0,175,255,1.00)",
                        "direction": "ltr",
                        "fontFamily": "Roboto",
                        "fontSize": "16px",
                        "fontWeight": "500",
                        "justifyContent": "center",
                        "letterSpacing": "0px",
                        "lineHeight": "22px",
                        "msFlexAlign": "center",
                        "msFlexPack": "center",
                        "paddingTop": "1px",
                        "textAlign": "center",
                        "textDecoration": "none",
                        "textTransform": "uppercase",
                      }
                    }
                  >
                    Request specific amount
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "height": "16px",
            }
          }
        />
        <div
          className="css-view-1dbjc4n"
          data-gdtype="copybutton"
          style={
            Object {
              "WebkitAlignItems": "stretch",
              "WebkitBoxAlign": "stretch",
              "WebkitBoxPack": "center",
              "WebkitJustifyContent": "center",
              "alignItems": "stretch",
              "backgroundColor": "rgba(0,175,255,1.00)",
              "borderBottomColor": "rgba(0,175,255,1.00)",
              "borderBottomLeftRadius": "50px",
              "borderBottomRightRadius": "50px",
              "borderBottomStyle": "solid",
              "borderBottomWidth": "0px",
              "borderLeftColor": "rgba(0,175,255,1.00)",
              "borderLeftStyle": "solid",
              "borderLeftWidth": "0px",
              "borderRightColor": "rgba(0,175,255,1.00)",
              "borderRightStyle": "solid",
              "borderRightWidth": "0px",
              "borderTopColor": "rgba(0,175,255,1.00)",
              "borderTopLeftRadius": "50px",
              "borderTopRightRadius": "50px",
              "borderTopStyle": "solid",
              "borderTopWidth": "0px",
              "boxShadow": "0px 0.75px 1.5px rgba(0,0,0,0.24)",
              "display": "flex",
              "justifyContent": "center",
              "marginBottom": "0px",
              "marginLeft": "0px",
              "marginRight": "0px",
              "marginTop": "0px",
              "minHeight": "44px",
              "minWidth": "64px",
              "msFlexAlign": "stretch",
              "msFlexPack": "center",
              "paddingBottom": "0px",
              "paddingLeft": "0px",
              "paddingRight": "0px",
              "paddingTop": "0px",
            }
          }
        >
          <div
            className="css-cursor-18t94o4 css-view-1dbjc4n"
            data-focusable={true}
            disabled={false}
            onKeyDown={[Function]}
            onKeyPress={[Function]}
            onKeyUp={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            role="button"
            style={
              Object {
                "borderBottomLeftRadius": "50px",
                "borderBottomRightRadius": "50px",
                "borderTopLeftRadius": "50px",
                "borderTopRightRadius": "50px",
                "cursor": "pointer",
                "msTouchAction": "manipulation",
                "overflowX": "hidden",
                "overflowY": "hidden",
                "position": "relative",
                "touchAction": "manipulation",
              }
            }
            tabIndex="0"
          >
            <div
              className="css-view-1dbjc4n"
              style={
                Object {
                  "WebkitAlignItems": "center",
                  "WebkitBoxAlign": "center",
                  "WebkitBoxDirection": "normal",
                  "WebkitBoxOrient": "horizontal",
                  "WebkitBoxPack": "center",
                  "WebkitFlexDirection": "row",
                  "WebkitJustifyContent": "center",
                  "alignItems": "center",
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "msFlexAlign": "center",
                  "msFlexDirection": "row",
                  "msFlexPack": "center",
                }
              }
            >
              <div
                className="css-text-901oao css-textOneLine-bfa6kz"
                dir="auto"
                style={
                  Object {
                    "color": "rgba(255,255,255,1.00)",
                    "direction": "ltr",
                    "fontFamily": "Roboto-Medium, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                    "letterSpacing": "1px",
                    "marginBottom": "9px",
                    "marginLeft": "16px",
                    "marginRight": "16px",
                    "marginTop": "9px",
                    "textAlign": "center",
                  }
                }
              >
                <div
                  className="css-view-1dbjc4n"
                  style={
                    Object {
                      "WebkitAlignItems": "center",
                      "WebkitBoxAlign": "center",
                      "WebkitBoxDirection": "normal",
                      "WebkitBoxFlex": 1,
                      "WebkitBoxOrient": "horizontal",
                      "WebkitFlexBasis": "0%",
                      "WebkitFlexDirection": "row",
                      "WebkitFlexGrow": 1,
                      "WebkitFlexShrink": 1,
                      "alignItems": "center",
                      "display": "flex",
                      "flexBasis": "0%",
                      "flexDirection": "row",
                      "flexGrow": 1,
                      "flexShrink": 1,
                      "msFlexAlign": "center",
                      "msFlexDirection": "row",
                      "msFlexNegative": 1,
                      "msFlexPositive": 1,
                      "msFlexPreferredSize": "0%",
                    }
                  }
                >
                  <span
                    className="css-text-901oao css-textHasAncestor-16my406"
                    dir="auto"
                    style={
                      Object {
                        "WebkitAlignItems": "center",
                        "WebkitBoxAlign": "center",
                        "WebkitBoxPack": "center",
                        "WebkitJustifyContent": "center",
                        "alignItems": "center",
                        "color": "rgba(255,255,255,1.00)",
                        "direction": "ltr",
                        "fontFamily": "Roboto",
                        "fontSize": "16px",
                        "fontWeight": "500",
                        "justifyContent": "center",
                        "letterSpacing": "0px",
                        "lineHeight": "22px",
                        "msFlexAlign": "center",
                        "msFlexPack": "center",
                        "paddingTop": "1px",
                        "textAlign": "center",
                        "textDecoration": "none",
                        "textTransform": "uppercase",
                      }
                    }
                  >
                    Receive via wallet link
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
